cookbooks/aws-parallelcluster-environment/recipes/config/efs.rb

```diff
@@ -15,14 +15,16 @@
 id_array = node['cluster']['efs_fs_ids'].split(',')
 encryption_array = node['cluster']['efs_encryption_in_transits'].split(',')
 iam_array = node['cluster']['efs_iam_authorizations'].split(',')
+accesspoint_id_array = node['cluster']['efs_accesspoint_ids'].split(',')
 
 # Identify the previously mounted filesystems and remove them from the set of filesystems to mount
 shared_dir_array.each_with_index do |dir, index|
-  next unless node['cluster']['internal_shared_dirs'].include?(dir) || dir == "/home" || dir == "home" || dir == node['cluster']['internal_initial_shared_dir']
+  next unless node['cluster']['internal_shared_dirs'].include?(dir) || dir == node['cluster']['internal_initial_shared_dir']
```

Comment:
why did you remove the condition on `/home` and `home`?

Reply:
It would skip mounting /home if it was specified in the SharedStorage settings - which is the main use case in our project. Is this a problem? From the underlying issue I thought it was not with PC 3.x

Reply:
This does not skip mounting `/home` when its specified in shared storage.  That mounting is done elsewhere, since /home is a special case for mounted file systems

Reply:
np!